packages/types/src/clerk.ts

```diff
@@ -73,7 +73,7 @@ export interface Clerk {
    * @experimental
    * Clerk Satellite Frontend API string.
    */
-  domain?: string;
+  domain?: string | ((url: URL) => string);
```

Comment:
❓ shouldn't there be an `isSatellite?` here ?

Reply:
@dimkl I think the question should be, do we still need domain there 🤔?